src/convert_prompt.py

```diff
@@ -39,15 +39,19 @@
 
 def get_dataset(dataset_name):
     dataset = load_dataset(dataset_name, split="train")
+    cap = min(6500, len(dataset))
```

Comment:
is it possible to make this an optional parameter? 
Basically, we don't want to sample by default, unless `--sample 6500` is given in the input.

Reply:
Ya, I'll fix it now.